agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -149,7 +151,22 @@ def add_variant(
         else:
             click.echo(click.style(f"Error while adding variant: {ex}", fg="red"))
         return None
+
+    # Get user profile
+    user_id = client.retrieve_user_id(host, api_key)
     if overwrite:
+        # Track a deployment event
+        if tracking_enabled:
+            event_track.capture_event(
+                user_id,
+                "app_deployment",
+                body={
+                    "app_id": app_id,
+                    "deployed_by": user_id,
+                    "environment": "CLI",
```

Comment:
Maybe add source: "oss" or "cloud" depending on whether the api_key is null or not

```suggestion
                    "environment": "CLI",
                    "version": "cloud" if api_key else "oss"
```

Reply:
Thank you!